src/test-run/commands/assertion.js

```diff
@@ -29,29 +29,20 @@ function initAssertionParameter (name, val, skipVisibilityCheck) {
 }
 
 // Commands
-export default class AssertionCommand extends Assignable {
-    constructor (obj) {
-        super(obj);
+export default class AssertionCommand extends CommandBase {
+    constructor (obj, testRun) {
```

Comment:
Why did we remove the class members initializing in constructor? The dynamic defining of class members works slower, because of the v8 hidden classes implementation. I believe it's a good practice to define class members with default values in constructor body always.

Reply:
Because a command does not have it own properties. It is assignable object that assign and initialize all properties from the source and does not know about default values. I want to exclude the case we will have different properties and its default values in the source object and in the constructed command.